test/unit/app/index-tests.ts

```diff
@@ -31,143 +28,145 @@ var boilerplateFiles = [
-            sandbox.stub(yeoman.Base.prototype, yoDestinationPathFunctionName, function () {
+        test('Should not init a new git repository when the destination directory already has a git repo initialized', (done) => {
+            // this stub is to ensure that the tmp directory (see below) creates the .git directory in
+            // the same directory as the destinationRoot of the generator.
+            sandbox.stub(yeoman.Base.prototype, yoDestinationPathFunctionName, () => {
                 return path.join(process.cwd(), baseAppName);
             });
 
             helpers.run(generatorRoot)
-                .inTmpDir(function (dir) {
+                .inTmpDir((dir) => {
                     fs.mkdirSync(path.join(path.resolve(dir), '.git'));
                 })
                 .withPrompts({
                     appName: baseAppName,
                     description: appDescription,
-                    type: inputConfig.boilerplatePromptValue,
+                    type: inputConfig.boilerplatePromptValue
                 })
                 .toPromise()
-                .then(function () {
+                .then(() => {
                     assert.deepEqual(gitInitCommandSpy.called, false);
                     done();
                 });
         });
-
-        test('Should create and scaffold into a new directory if the specified app name differs from the current directory with the Base option', function (done) {
-            helpers.run(generatorRoot)
-                .withPrompts({
-                    appName: baseAppName,
-                    description: appDescription,
-                    type: inputConfig.boilerplatePromptValue,
-                })
-                .toPromise()
-                .then(function (dir) {
-                    assert.equal(path.basename(process.cwd()), baseAppName);
-                    assert.equal(path.resolve(process.cwd()), path.join(dir, baseAppName));
-                    done();
+        // tslint:disable-next-line:max-line-length
+        test('Should create and scaffold into a new directory if the specified app name differs from the current directory with the Base option',
+            (done) => {
+                helpers.run(generatorRoot)
+                    .withPrompts({
+                        appName: baseAppName,
+                        description: appDescription,
+                        type: inputConfig.boilerplatePromptValue
+                    })
+                    .toPromise()
+                    .then((dir) => {
+                        assert.equal(path.basename(process.cwd()), baseAppName);
+                        assert.equal(path.resolve(process.cwd()), path.join(dir, baseAppName));
+                        done();
+                    });
+            });
+        // tslint:disable-next-line:max-line-length
+        test('Should scaffold into the current directory when the specified app name matches the current directory name with the Base option',
```

Comment:
What about breaking the string up into multiple lines? That's what I've typically done w/long test names